rakelib/utils.rake

```diff
@@ -97,14 +97,12 @@ class Utils
 
   # run a command, pipe output through 'xcpretty' and store the output in CI artifacts
   def self.xcpretty(cmd, task, subtask)
-    name = (task.name + (subtask.empty? ? '' : "_#{subtask}")).gsub(/[:-]/, '_')
     command = [*cmd].join(' && ')
 
     if ENV['CI']
-      Rake.sh "set -o pipefail && (#{command}) | tee \"#{ENV['CIRCLE_ARTIFACTS']}/#{name}_raw.log\" | " \
-        "bundle exec xcpretty --color --report junit --output \"#{ENV['CIRCLE_TEST_REPORTS']}/xcode/#{name}.xml\""
+      Rake.sh %(set -o pipefail && (#{command}) | bundle exec xcpretty --color --report junit)
```

Comment:
I'm quite new to GHA, how are raw logs and artefacts handled? Does that change mean we won't be able to access the raw logs from the GitHub Action UI?

Reply:
Few things:
- We actually never really used those artifacts? Point of them was to be able to maybe analyse them in some follow-up step
- You can simply see **all** the output of a job/step right here in GitHub, there's no extra output

Reply:
There's a difference between the output and the artefacts though. The output is `xcpretty`-fied, while the point of having the artefacts in CircleCI is because – thanks to `tee` being inserted before `xcpretty` – the files we stored in the artefacts contained the raw logs (much more verbose) as opposed to the short/prettified ones that were printed on stdout. And I just checked and GitHub Action seems to support artefacts as well (actually when you click on a check on a pull request to see the details you can then click again on a specific job run and see its artefacts if it had any).

Tbh the point of those raw outputs is that if there's something wrong, sometimes the prettified output is not verbose enough to understand the real underlying raw error spit out by Xcode, and being able to look at the raw logs could help.  But in practice indeed I don't think we used those raw logs at any point – especially for SSK – so probably not reproducing the behavior there anyway at least for now.

Reply:
We can easily add them back at some (later) point if we really want/need them 🤷

Reply:
Exactly